00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,593 @@
+Changes Requested indicator:
+
+![PR workflow diagram](request-rereview.png)
+
+This is represented by the `request_rereview` state
+in the flowchart. From here, automation again helps the process along:
+
+- If the author does not have any other outstanding reviews, they are unassigned
+  from the PR.
+- The reviewer that submitted the now-addressed review is re-assigned, and the
+  PR is moved back to the `reviewer_assigned` state.
+  
+At this point, the cycle is complete, and it repeats from the beginning until
+the reviewer is satisfied with the PR and submits their approval.
+
+### Merge criteria
+
+With a clear framework in place for reviewing and approving pull requests, it's
+now more feasible for the Enarx project to begin enforcing stricter merge
+requirements.
+
+A pull request should only be merged when:
+
+- all required code tests pass
+- it has two or more approving reviews
+- every reviewer that has submitted a review has updated their review to
+  "Approved"
+
+The latter exists to ensure that all feedback from all reviewers is addressed
+before merge.
+
+With concrete criteria in place, automated merging, a longer-term automation
+goal, may become feasible. This is represented by the final green arrow from
+`approved` to `merged` in the flowchart.
+
+## Drawbacks
+
+- The team will need to attend a few more meetings. The daily standups, as
+  currently designed, lose their focus on individual areas of the project per
+  day; this was useful in case an external contributor was interested in one
+  specific area of the project. However, we hope the added free-form discussion
+  time will allow for relevant questions to be asked and talked about anyway.
+
+- Implementing some of the automation exactly as specified may end up being a
+  challenge. There's further discussion of this below.
+
+## Rationale and alternatives
+
+There are many ways to approach sprints, and we can't claim this is the best
+possible sprint methodology out there; only time will tell if this works for us
+once we start using it. Additionally, this process is not set in stone; anything
+can change from one sprint to the next. However, there are a few nice
+characteristics worth highlighting:
+
+- Enarx is still a fairly small team, and we like some flexibility in our
+  process. The modular design of our board supports that -- for example, if we
+  consider a new issue or pull request particularly important, we can manually
+  assign it and have it skip the backlog entirely.
+- Moving to a single, organization-level board allows visitors to see a snapshot
+  of what's currently a top priority for Enarx.
+- With automation, this process has potential to minimize the busywork that
+  project developers have to deal with, and only require human judgment when it
+  is needed.
+- Running through a triage every day ensures issues are promptly reviewed by the
+  team after they are submitted.
```

Comment:
I have not seen been done before, but i like the idea. I'm looking forward to seeing how this works out, as i think the idea of spreading out a backlog grooming, doing a bit every day, is quite smart.